versions.tf

```
@@ -15,43 +15,23 @@
  */
 
 terraform {
-  required_version = ">= 0.13.0"
+  required_version = "~> 1.0.0"
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 3.42"
-    }
-    google-beta = {
-      source  = "hashicorp/google-beta"
-      version = "~> 3.42"
+      version = "3.72.0"
     }
     helm = {
       source  = "hashicorp/helm"
-      version = "~> 1.2.0"
+      version = "~> 2.2.0"
```

lets pin these to minor versions moving forward
```suggestion
      version = "~> 2.2"
```

---

Fixed!